app/Console/Commands/WaitingLists/CheckMembersMeetActivityRules.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Console\Commands\WaitingLists;
+
+use App\Console\Commands\Command;
+use App\Events\Training\AccountMarkedForRemovalFromWaitingList;
+use App\Events\Training\AccountRegainedActivityRequirementsForWaitingList;
+use App\Models\NetworkData\Atc;
+use App\Models\Training\WaitingList;
+use Carbon\Carbon;
+
+class CheckMembersMeetActivityRules extends Command
+{
+    protected $minutesRequired = 720; // 12 hours is represented as 720 minutes
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'waitinglists:checkmembersmeetactivityrules';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Checks ATC waiting list activity rules are being met for each account';
+
+    /**
+     * Executes all necessary console commands.
+     */
+    public function handle()
+    {
+        WaitingList::all()->filter(function ($waitingList) {
+            return $waitingList->isAtcList();
+        })->each(function (WaitingList $waitingList) {
+            $waitingList->accounts
+            ->filter(function ($account) {
+                return $account->pivot->current_status->name == 'Active';
+            })->each(function ($account) use ($waitingList) {
+                $activeMinutes = Atc::where('account_id', $account->id)
+                    ->whereDate('disconnected_at', '>=', Carbon::parse('3 months ago'))
+                    ->isUk()
```

Comment:
```suggestion
                    ->isUk()
                    ->whereDate('disconnected_at', '>=', Carbon::parse('3 months ago'))
```

Filtering down the amount of rows you need to filter on `disconnected_at` since I don't recall there being an index on that access pattern of columns.